packages/melos/lib/src/commands/bootstrap.dart

```diff
@@ -363,7 +371,13 @@ mixin _BootstrapMixin on _CleanMixin {
     // dependencies that have a different version specified in the workspace.
     final dependenciesToUpdate = workspaceDependencies.entries.where((entry) {
       if (!packageDependencies.containsKey(entry.key)) return false;
-      if (packageDependencies[entry.key] == entry.value) return false;
+      // TODO: We may want to replace the `pubspec` dependency with something
```

Comment:
It is replaced with pubspec_parse now

Reply:
Is this PR obsolete or does the big still exist? I am updating it, but I realized that the package has changed and I am not sure if it solves the problem by itself.

Reply:
I'm not sure. Can you try running `main`?